pkg/handler/server/client.go

```diff
@@ -137,6 +157,27 @@ func (c *Client) Get(ctx context.Context, organizationID, projectID, serverID st
 
 // Update a resource.
 func (c *Client) Update(ctx context.Context, organizationID, projectID, identityID, serverID string, request *openapi.ServerWrite) (*openapi.ServerRead, error) {
+	identity, err := identity.New(c.client, c.namespace).GetRaw(ctx, organizationID, projectID, identityID)
+	if err != nil {
+		return nil, err
+	}
+
+	provider, err := region.NewClient(c.client, c.namespace).Provider(ctx, identity.Labels[constants.RegionLabel])
+	if err != nil {
+		return nil, errors.OAuth2ServerError("failed to create region provider").WithError(err)
+	}
+
+	// REVIEW_ME: Is the `unikorn:organization:id` property immutable on the image?
```

Comment:
For now yes, I don't see a world where you'd transfer between orgs.  You can always re-upload!